docs/3_guides/0_submitting_proofs.md

```diff
@@ -68,16 +70,16 @@ To use it, you can use the `aligned` CLI, as shown with the following example:
 
 ```bash
 aligned deposit-to-batcher \
---rpc_url https://ethereum-holesky-rpc.publicnode.com \
---network holesky \
+--rpc_url https://ethereum-rpc.publicnode.com \
```

Comment:
Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error